mods/gennext/README.md

```diff
@@ -236,6 +237,14 @@ New mechanic: Intrinsics:
 
   - Mismagius: Cursed Body
 
+  - Rotom: Motor Drive plus a hidden ability based on forme:
+    - Normal: Adaptability
+    - Heat: Flash Fire
+    - Wash: Storm Drain
+    - Mow: Sap Sipper
+    - Fan: Gale Wings
+    - Frost: Snow Warning
```

Comment:
~~Says the guy who gave Gengar freakin' Cursed Body, especially after the buff~~

I can give it Hydration. So we're looking at one ability per forme?

Why does Flygon have a hidden ability in its intrinsics and nobody else does